landing page first edition

BetterSol opened the pull request.

dsfreedom reviewed (approved):
Incredible work!!! As for this much big work very few minor comments. I approve but still better to change those few things.

BetterSol commented on src/index.html:
```diff
@@ -1,13 +1,324 @@
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link
```
yes, it was work-in-progres :)

BetterSol commented on src/index.html:
```diff
@@ -1,13 +1,324 @@
+                <a class="nav__link" href="#move-free">About us</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="compare">Compare bikes</a>
```
didn't notice that. Thank you so much for your attentiveness!

superpooperxxx reviewed (changes requested):
hey, good job🔥

1. Hero section should be 100vh
<img width="864" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/f2d6c919-284a-4608-8702-96db76323681">

2. Same here
<img width="864" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/b19de7ee-8d01-4ff0-8425-dbb97b48e5be">

3. Wrong heading
<img width="530" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/b99fb997-ccb1-40ce-ba83-79679d794661">

4. Looks like line height here is incorrent
<img width="481" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/8ae120ce-4731-4ea4-a125-04b627b3df54">

5. Check the styles for heading + you can also make it a link (mailto)
<img width="98" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/2859f7e9-c945-44a2-9459-179b83c3ad33">

6. Did you do this container on purpose ?
<img width="634" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/6a217e7a-c953-43bc-8ae4-e92f97084417">

7. Mobile looks good except this
<img width="220" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/e160ec4f-64ad-465c-a81b-5f6fc0e25e61">

8. 🚨 Optimise images using some compressor like TinyPNG

BetterSol commented:
Hey, thanks for reviewing my landing page! Fixed all elements according to your suggestions. Still working on a part with compressing images though. Trying to do my best:)

polosanya reviewed (changes requested):
Keep up the excellent work! Here are a few moments that would be great to fix:
1. Hero section still doesn't look correct on large screens.
Also, a horizontal scroll is present for some reason
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/a3778f1f-93d3-4c02-9d40-5d89eb103a16)
2. Same with the burger menu
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/efa61dd4-498d-4120-a512-cf0ae70dc578)
3. The button is too wide. Also, would be great to navigate somewhere on click
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/9d440e9a-2737-429c-b791-7137cb5fc274)
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/603c6680-06e8-4843-8670-9245e82f8275)
4. Button's description doesn't match the design
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/260a8986-c8ff-44e6-bf99-c70cab5acde8)
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/dac4d91a-19f9-4c63-bcb8-8068c66b03b1)
Also, it should be a cursor pointer on hover
5. Pay attention that second word should be in upper case
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/9881c4fc-3aee-429d-a085-215cc2c1c752)
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/62955f7c-1bda-4eaf-9de6-0abfdb6637e5)

BetterSol commented:
Thank you! I fixed all the elements you suggested. I still have some issues with the button font, though I've implemented styles according to the design.

superpooperxxx reviewed (approved):
You are 1 step from perfection🔥

1.Make hero section always be 100vh (give `header__content` class `height: 100vh`)
<img width="864" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/9e5b9167-8ff1-4e85-9a0b-8fb00b82c488">

2. The same width navigation
<img width="863" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/2387b90a-226f-4eae-b720-dd6471630198">

✅
<img width="863" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/de20140f-832f-43ff-a1bd-bf93c22c1dc8">
<img width="864" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/49764881/45280da6-cfbc-4dd3-b593-49d0dad7181b">
